src/braket/circuits/free_parameter.py

```diff
@@ -0,0 +1,79 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from numbers import Number
+
+from sympy import Symbol
+
+from braket.circuits.free_parameter_expression import FreeParameterExpression
+
+
+class FreeParameter(FreeParameterExpression):
+    """
+    Class 'FreeParameter'
```

Comment:
More description here

Reply:
Added

Reply:
I mean to put the description here, in the class-level docstring rather than `__init__`. Same goes for FreeParameterExpression.